src/main/scala/com/lightbend/sbt/javaagent/JavaAgentPackaging.scala

```diff
@@ -21,11 +21,14 @@ object JavaAgentPackaging extends AutoPlugin {
 
   override def requires = JavaAgent && PluginRef("com.typesafe.sbt.packager.archetypes.JavaAppPackaging")
 
-  override def projectSettings = Seq(
-    mappings in Universal ++= agentMappings.value.map(m => m._1 -> m._2),
-    bashScriptExtraDefines ++= agentBashScriptOptions.value,
-    batScriptExtraDefines ++= agentBatScriptOptions.value
-  )
+  override def projectSettings = {
+    import com.typesafe.sbt.packager.Keys
+    Seq(
+      mappings in Universal ++= agentMappings.value.map(m => m._1 -> m._2),
+      Keys.bashScriptExtraDefines ++= agentBashScriptOptions.value,
+      Keys.batScriptExtraDefines ++= agentBatScriptOptions.value
```

Comment:
Yes, definitely. I just assumed that they where auto imported here, but of course they couldn't be since the loading is via reflection. Thanks for the catch!